e2e/cypress/integration/channel/mobile_profile_popover_spec.js

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Profile popover', () => {
+    before(() => {
+        // # Go to the top public channel with "user-1"
+        cy.apiLogin('user-1');
+        cy.toTopPublicChannelView();
+        cy.viewport('iphone-6');
+    });
+
+    it('M18715 Profile popover should render', () => {
+        cy.getLastPostId().then((postId) => {
+            // add wait time to ensure image is rendered and can be clicked
+            cy.wait(TIMEOUTS.TINY);
+
+            // # Click on user profile image
+            cy.get(`#post_${postId}`).find('.profile-icon > img').click();
```

Comment:
This fails when message display setting is set to compact mode.  Please explicitly set it to standard after user-1 has logged in.
Bonus: Not required but would be nice if you can add case to open profile popover on compact mode (that is by clicking on username)

Reply:
Oh that's a good idea, will definitely add those in :)